### ISSUE TITLE
is strong attack z or y?

### ISSUE THREAD

**Issue by anxuthu** [amacati/SoulsGym#47]
[here](https://soulsgym.readthedocs.io/en/latest/getting_started/setup.html) in the doc, strong attack is mapped to keyboard 'z', but in [keys.yaml](https://github.com/amacati/SoulsGym/blob/3c4b69d02d5e78dc375703aaedcd4d56244f25af/soulsgym/core/data/keys.yaml#L15), it's mapped to 'y'.

Would this be an issue?

**Comment by amacati** [comment]
Good catch ;) I am German, and on the German keyboard layout, z and y are swapped. With my setup, everything works as intended, but if you change the layout to qwerty, you won't be able to do heavy attacks. 

I should probably add a keyboard check on initialization or change the key to a character that is consistent across both layouts. I think it's easier to go for the latter solution because of the amount of layouts that are out there.
